user_tools/src/spark_rapids_tools/tools/top_candidates.py

```diff
@@ -85,12 +86,21 @@ def _generate_output_table_internal(self, output_df: pd.DataFrame) -> pd.DataFra
         """
         Internal implementation to prepare the output table. This can be overridden by the child classes.
         """
+        # Append 'Speedup Category Order' and 'Total Core Seconds' columns to output_df for sorting order
+        # First, sort based on speedup category (decs order). Create and append 'Speedup Category Order' column.
+        speedup_category_order = ['Not Recommended', 'Small', 'Medium', 'Large']
```

Comment:
we read the categories from the yaml file. It will be nice to avoid hardcoding them if possible.
We can actually define a new Enum for the categories (if it is not already defined) instead of doing that in the middle of an unrelated method.

Reply:
fixed, thanks!